wallet/walletrpc.c

```diff
@@ -718,6 +742,11 @@ static struct command_result *json_signpsbt(struct command *cmd,
 		return command_fail(cmd, LIGHTNINGD,
 				    "No wallet inputs to sign");
 
+	// Update the keypaths on any outputs that are in our wallet (change addresses).
+	res = match_psbt_outputs_to_wallet(psbt, cmd->ld->wallet);
```

Comment:
I don't see a codepath that returns non-NULL from the callee.

Reply:
fixed